common/controllers/form-wizard.js

```diff
@@ -1,16 +1,37 @@
 const Sentry = require('@sentry/node')
 const { Controller } = require('hmpo-form-wizard')
-const { map, fromPairs } = require('lodash')
+const { map, fromPairs, forEach, mapKeys } = require('lodash')
 
 const fieldHelpers = require('../helpers/field')
 
 class FormController extends Controller {
   middlewareSetup() {
     super.middlewareSetup()
+    this.use(this.setupAddMultipleFields)
     this.use(this.setupConditionalFields)
     this.use(this.setFieldContext)
   }
 
+  middlewareActions() {
+    super.middlewareActions()
```

Comment:
What's the point of just calling the `super` method?

Reply:
None, I forgot to remove this after changing the above...